pkg/env/costmodelenv.go

```diff
@@ -23,6 +23,8 @@ const (
 	AzureOfferIDEnvVar                   = "AZURE_OFFER_ID"
 	AzureBillingAccountEnvVar            = "AZURE_BILLING_ACCOUNT"
 	AzureDownloadBillingDataToDiskEnvVar = "AZURE_DOWNLOAD_BILLING_DATA_TO_DISK"
+	AzureResourceGroupEnvVar             = "AZURE_RESOURCE_GROUP_NAME"
+	AzureClusterNameEnvVar               = "AZURE_CLUSTER_NAME"
```

Comment:
would it be possible to get these values from the kubernetes api without having them configured. At a glance the node label `kubernetes.azure.com/cluster` and the providerID seem to hold the relevant values, although there may be formatting challenges .

Reply:
Thanks for the suggestion. Fixed it.

Reply:
@Sean-Holcomb Underscore is allowed in both resource group name and cluster name. So, it is not possible to parse using node labels. Can you please share another way to do it?